scipy/signal/signaltools.py

```diff
@@ -1896,6 +1896,9 @@ def resample_poly(x, up, down, axis=0, window=('kaiser', 5.0)):
         Desired window to use to design the low-pass filter. See
         `scipy.signal.get_window` for a list of windows and required
         parameters.
+    num : array_like, optional
+         FIR filter coefficients of the low-pass filter to be used. Overrides
+         the `window` argument if not None.
```

Comment:
Just roll this into `window`. See e.g. how `resample` allows multiple types for `window`:

http://docs.scipy.org/doc/scipy-0.16.0/reference/generated/scipy.signal.resample.html